applications/feedback_sentiment_analyzer/cdk/lib/functions.ts

```diff
@@ -164,11 +164,85 @@ const JAVASCRIPT_FUNCTIONS = [
   },
 ];
 
+const JAVA_BUNDLING_CONFIG = {
+  command: [
+    "/bin/sh",
+    "-c",
+    "mvn install && cp /asset-input/target/creating_fsa_app-1.0-SNAPSHOT.jar /asset-output/",
+  ],
+  outputType: BundlingOutput.ARCHIVED,
+  user: "root",
+  image: Runtime.JAVA_11.bundlingImage,
+  volumes: [
+    {
+      hostPath: `${process.env["HOME"]}/.m2/`,
+      containerPath: "/root/.m2",
+    },
+  ],
+ 
+};
+
+const JAVA_FUNCTIONS: AppFunctionConfig[] = [
+  {
+    ...BASE_APP_FUNCTION,
+    name: "ExtractText",
+    codeAsset: () => {
+      const source = resolve(
+        "../../../javav2/usecases/creating_fsa_app"
+      );
+      return Code.fromAsset(source, {
+        bundling: JAVA_BUNDLING_CONFIG,
+      });
+    },
+    handler: "com.example.fsa.handlers.S3Handler"
+  },
```

Comment:
Since the `codeAsset` is the same for all four of these functions, this can be simplified.

```suggestion
const BASE_JAVA_FUNCTION = {
 ...BASE_APP_FUNCTION,
 codeAsset: () => {
      const source = resolve(
        "../../../javav2/usecases/creating_fsa_app"
      );
      return Code.fromAsset(source, {
        bundling: JAVA_BUNDLING_CONFIG,
      });
    },
};

const JAVA_FUNCTIONS: AppFunctionConfig[] = [
  {
    ...BASE_JAVA_FUNCTION,
    name: "ExtractText",
    handler: "com.example.fsa.handlers.S3Handler"
  },
```

Reply:
I worked with different combinations. The current format works and craates Java 11 runtime Lambda functions.